Cargo.toml

```diff
@@ -38,57 +38,58 @@ drop_down = []
 sidebar = []
 
 default = [
-    "badge",
-    "card",
-    "number_input",
-    "date_picker",
-    "color_picker",
-    "grid",
+    # "badge",
+    # "card",
+    # "number_input",
+    # "date_picker",
+    # "color_picker",
+    # "grid",
     "tab_bar",
-    "tabs",
-    "time_picker",
-    "slide_bar",
-    "wrap",
-    "selection_list",
-    "quad",
-    "context_menu",
-    "spinner",
-    "drop_down",
-    "menu",
-    "sidebar",
+    # "tabs",
+    # "time_picker",
+    # "slide_bar",
+    # "wrap",
+    # "selection_list",
+    # "quad",
+    # "context_menu",
+    # "spinner",
+    # "drop_down",
+    # "menu",
+    # "sidebar",
 ]
 
 [dependencies]
 cfg-if = "1.0"
 chrono = { version = "0.4.39", optional = true, features = ["wasmbind"] }
 getrandom = { version = "0.3.1" }
-iced_fonts = "0.1.1"
 itertools = { version = "0.14.0", optional = true }
 num-format = { version = "0.4.4", optional = true }
 num-traits = { version = "0.2.19", optional = true }
 web-time = "1.1.0"
 
 [dependencies.iced]
-#git = "https://github.com/iced-rs/iced.git"
-#rev = "b474a2b7a763dcde6a377cb409001a7b5285ee8d"
-version = "0.13.1"
+version = "0.14.0-dev"
+git = "https://github.com/iced-rs/iced.git"
+# rev = "beddf49c"
```

Comment:
you could try branch = main too

Reply:
I prefer you to comment out version and rev and add branch = "main"